plugin/advisor/schema_review.go

```diff
@@ -156,37 +156,46 @@ type RequiredColumnRulePayload struct {
 }
 
 // UnamrshalNamingRulePayloadAsRegexp will unmarshal payload to NamingRulePayload and compile it as regular expression.
-func UnamrshalNamingRulePayloadAsRegexp(payload string) (*regexp.Regexp, error) {
+func UnamrshalNamingRulePayloadAsRegexp(payload string) (*regexp.Regexp, int, error) {
 	var nr NamingRulePayload
 	if err := json.Unmarshal([]byte(payload), &nr); err != nil {
-		return nil, fmt.Errorf("failed to unmarshal naming rule payload %q: %q", payload, err)
+		return nil, 0, fmt.Errorf("failed to unmarshal naming rule payload %q: %q", payload, err)
 	}
+
+	if nr.MaxLength <= 0 {
```

Comment:
If I don't want to set the max length rule, what should I do?

Reply:
Good question🤔 Do you think we need to support the negative number which means skipping the length check?

Reply:
SGTM~